packages/cloud_firestore/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.9.7
+
+* Fixes an iOS regression in getDocuments introduced in 0.9.6.
```

Comment:
Would be useful to describe what was the regression (both here and in the PR description)